packages/Filters/src/Filters.js

```diff
@@ -0,0 +1,148 @@
+  const filtersRef = React.useRef(null);
+  const [isOpen, setIsOpen] = React.useState(false);
+
+  function handleClickTrigger() {
+    setIsOpen(prevIsOpen => !prevIsOpen);
+  }
+
+  function handleApply() {
+    onApply();
+    setIsOpen(false);
+  }
+
+  function handleCancel() {
+    setIsOpen(false);
+    onCancel();
+  }
+
+  function handleClear() {
+    onClear();
+  }
+
+  return (
+    <FilterContext.Provider value={{ filtersRef, columns, data, operator, onChangeOperator, rulesByType }}>
+      <Panel data-pka-anchor="filters.panel" isCompact isOpen={isOpen} onClose={handleCancel}>
+        <Panel.Header>
+          <sc.PanelHeaderWrapper>
+            <FilterIcon aria-hidden="true" />
+            {I18n.t("filters.label")}
+          </sc.PanelHeaderWrapper>
+        </Panel.Header>
+
+        <Panel.Trigger
+          data-pka-anchor="filters.panelTrigger"
+          icon={<FilterIcon />}
+          isSemantic={false}
+          kind={Button.types.kind.FLAT}
+          onClick={handleClickTrigger}
+        >
+          {getLabelText(appliedNumber, I18n)}
+        </Panel.Trigger>
+        <sc.FiltersPanel ref={filtersRef}>
+          {React.Children.count(children) === 0 ? (
+            <sc.GenericNoAppliedPlaceholder>{I18n.t("filters.no_filters_applied")}</sc.GenericNoAppliedPlaceholder>
+          ) : (
+            children
+          )}
+          <Button data-pka-anchor="filters.addFilterButton" icon={<AddIcon />} onClick={onAddFilter}>
+            {I18n.t("filters.actions.add")}
+          </Button>
+        </sc.FiltersPanel>
+        <Panel.Footer isSticky>
+          <sc.Footer>
+            <Button data-pka-anchor="filters.applyButton" onClick={handleApply} kind="primary">
+              {I18n.t("actions.apply")}
+            </Button>
+            <Button data-pka-anchor="filters.cancelButton" onClick={handleCancel} kind="minor">
+              {I18n.t("actions.cancel")}
+            </Button>
+            <Button data-pka-anchor="filters.clearButton" onClick={handleClear} kind="minor">
+              {I18n.t("filters.actions.clear")}
+            </Button>
+          </sc.Footer>
+        </Panel.Footer>
+      </Panel>
+    </FilterContext.Provider>
+  );
+}
+
+Filters.types = {
+  operator: types.logicalFilterOperators,
+  rule: types.rules,
+  rulesByType: types.defaultRulesByType,
+};
+
+const propTypes = {
+  appliedNumber: PropTypes.number,
+  children: PropTypes.node,
+  columns: PropTypes.arrayOf(PropTypes.shape(columnShape)).isRequired,
+  data: PropTypes.arrayOf(PropTypes.shape({})),
+  onAddFilter: PropTypes.func.isRequired,
+  onApply: PropTypes.func.isRequired,
+  onCancel: PropTypes.func,
+  onChangeOperator: PropTypes.func,
+  onClear: PropTypes.func,
+  operator: PropTypes.oneOf([Filters.types.operator.AND, Filters.types.operator.OR]),
+
+  rulesByType: PropTypes.objectOf(PropTypes.arrayOf(PropTypes.objectOf(Filters.types.rule))),
+};
+
+const defaultProps = {
+  appliedNumber: 0,
```

Comment:
Small nit that i'm not adamant about, the proper english would be `numberApplied`